libs/pages/application/src/lib/feature/page-settings-terraform-configuration-feature/page-settings-terraform-configuration-feature.tsx

```diff
@@ -18,7 +23,11 @@ export default function PageSettingsTerraformConfigurationFeature() {
   const methods = useForm<TerraformGeneralData>({
     mode: 'onChange',
     defaultValues: match(service)
-      .with({ serviceType: 'TERRAFORM' }, (s) => ({ ...s, state: 'kubernetes' }))
+      .with({ serviceType: 'TERRAFORM' }, (s) => ({
+        ...s,
+        state: 'kubernetes',
```

Comment:
@fabienfleureau Could you remove that?
That's on me, it should have been removed before.